packages/swing-store/src/snapStore.js

```diff
@@ -87,14 +126,16 @@ export function makeSnapStore(
   /** @type {(input: string, f: NodeJS.ReadWriteStream, output: string) => Promise<void>} */
   async function filter(input, f, output) {
     const source = createReadStream(input);
-    const destination = createWriteStream(output);
+    const destination = createWriteStream(output, { flags: 'wx' });
```

Comment:
Why `x` ? Setting the executable bit doesn't make sense to me.

Reply:
`x` is for exclusive access. It will fail if the file already exists instead of truncating an existing file.